### DISCUSSION TITLE
Transfer project from Personal Account to Organisation

### DISCUSSION THREAD

**Discussion by PIMI-krl** [community/community#39826] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I'd like to transfer a project from a Personal Account to an Organisation

**Comment by ossanna16** [comment]
Hi there @PIMI-krl and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by CKtrace** [comment]
The information I know is below.

1. Enter the repository you want to move.
2. Click the Settings tab.
3. On the Repository Settings page, on the “Options” tab, at the bottom of the page there is an item called “Transfer ownership”. You can transfer storage ownership using the button in this section.

I hope my info is helpful!
Have a good day!

**Comment by AlexMi-Ha** [comment]
Go to any of your repositories
![image](https://user-images.githubusercontent.com/61502536/203807633-15651b97-37de-444c-a307-d16f39e68867.png)
Click on 'Settings'
Scroll Down and click on 'Transfer Ownership"
![image](https://user-images.githubusercontent.com/61502536/203807790-77558310-42f3-4269-b0f9-8112470dd1db.png)
 Finally enter your destination (Github user or organization) 
![image](https://user-images.githubusercontent.com/61502536/203807925-2612bb1e-f711-4827-a070-186bc603c5fc.png)

**Comment by PIMI-krl** [comment]
The issue is _not_ about transferring repositories as I've been able to do that just fine. What I'm struggling with is transfering a _project_ that I own on my _Personal Account_, even though I am a part of the organisation I am unable to transfer the _project_.

**Comment by evoludigit** [comment]
I have the same problem, transferring a repository did transfer the issues but not the projects referring to them.

**Comment by heribertolugo** [comment]
i am also in this scenario. but transferring from one personal account to another. as a workaround, i added the other account (to whom i transferred the repo to) as an admin. the other user can see the project but cannot access it. when clicking on the project it redirects to a 404 page.

**Comment by AhmedThahir** [comment]
Any update on this?

**Comment by Pictor13** [comment]
I also would expect to be able to transfer _all_ the tabs available in a repository (including "Projects") to an organization.  
A bit counterintuitive that it doesn't if you ask me; although I understand that Projects are not _dependencies_ of a repository but the other way around.

The use case is of about a dev that temporarily started a draft of a project. adding a repo for MVP, adding issues **and** organizing them on Kanban boards, but later on needing to transfer _all_ this work on a more appropriate location (an organization, that is the actual owner).  
While at the moment the drawback is having to redo the PM work and reorganize the boards by hand.

**Comment by mathieurousseau** [comment]
+1 I have moved my repo to the organization. But I can't move the project.

**Comment by max-gurr** [comment]
Also facing this issue - I want to transfer project ownership from my personal account to my organisation, or add the org as a collaborator so everyone in it can use the project.

**Comment by Antzed** [comment]
same here

**Comment by heribertolugo** [comment]
> i am also in this scenario. but transferring from one personal account to another. as a workaround, i added the other account (to whom i transferred the repo to) as an admin. the other user can see the project but cannot access it. when clicking on the project it redirects to a 404 page.
---

Update:

I'm wondering if any github personnel are aware of this thread. 

so i just noticed that the account i transferred the repo to (MySoftwareAdmin) and my personal account (heribertolugo) can see the project in question under the Projects tab in that repo. However the link points to https://github.com/users/MySoftwareAdmin/projects/1

which is incorrect. the working url is: https://github.com/users/heribertolugo/projects/1

but because of the transfer of the repo containing the project, the invalid url would logically be the correct one.

i can access the project if i click the links generated in the issues which are/were linked. i can also access it if i go into projects tab under my profile (opposed to the repo) at https://github.com/heribertolugo?tab=projects, however MySoftwareAdmin cannot see this project under projects tab in its profile.

even though both accounts have admin access on the project, i cannot see the project when i go into an issue and try to associate it with that project. it does not show in any of the 3 tabs (recent, repository, user) to link a project within the issue.

so the project appears in the repo's projects tab, but the link points to an incorrect location. in addition the project cannot be seen by an issue for attaching/linking.

i tried to rename it, to see if that fixes anything. i also tried to change the access. i also changed user privileges at "Manage Access". 

this seems like an easy fix. the new url created when transferring a repo is pointing to a resource that does not exist. the resource which it is attempting to point to is still residing at its original url. meanwhile ~~all~~ most links which pointed to the original are updated with the new (but invalid) url. just copy the resource to the new url and put a redirect in the old resource location to redirect to the new one. then figure out why it is not being listed in the list of projects which can be attached to an issue. it is still somehow linked to the actual repo, because it shows in projects tab.

**Comment by linus-guestrix** [comment]
I should of course be possible to transfer the ownership of a project as well as the ownership of individual repos. However, that does not seem to be the case. If anyone has managed to change ownership of a project, I'd be interested to hear about it.

**Comment by Annush73** [comment]
Привет

**Comment by Annush73** [comment]
> same here

**Comment by linus-guestrix** [comment]
I just found this script and tried it to migrate my project from one organisation to another, after I had already transfered ownership of the repos. And it worked! 
https://github.com/timrogers/gh-migrate-project

**Reply by pjpires** [reply]
> I just found this script and tried it to migrate my project from one organisation to another, after I had already transfered ownership of the repos. And it worked! [timrogers/gh-migrate-project](https://github.com/timrogers/gh-migrate-project)

This just worked for me too: had to manually re-create the custom information on my status fields as described in one of the steps, and lost one custom iteration field (which I wasn't able to re-create as it's impossible to enter dates in the past on a new one), but overall it worked way better than doing everything on my own. Thanks for the tip! 🙏

**Comment by Annush73** [comment]
- [ ] - **__**

**Comment by BlackBeltPanda** [comment]
How is this still not a feature?
